keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -287,9 +298,8 @@ CompletedIndexes: "2-4,6-7"
 The `kubectl describe` command crops the list of indexes if it's too long:
```

Comment:
sorry, I should've commented this in the original KEP PR, but IMO `2-4,6,7` is more readable than `2-4,6-7` (same number of chars). WDYT?